src/algebra/group/units.lean

```diff
@@ -128,6 +131,27 @@ section monoid
 
 end monoid
 
+section comm_monoid
+
+variables [comm_monoid α]
+
+theorem divp_eq_divp_of {x y : α} {ux uy : units α} (h : x * uy = y * ux) :
+  x /ₚ ux = y /ₚ uy :=
+calc x * (ux⁻¹ : units α) = x * uy * (uy⁻¹ : units α) * (ux⁻¹ : units α) :
+  by rw [units.mul_inv_cancel_right]
+                      ... = x * uy * (ux⁻¹ : units α) * (uy⁻¹ : units α) : mul_right_comm _ _ _
+                      ... = y * (uy⁻¹ : units α) : by rw [h, units.mul_inv_cancel_right]
+
+theorem divp_eq_divp_iff {x y : α} {ux uy : units α} :
+  x /ₚ ux = y /ₚ uy ↔ x * uy = y * ux :=
+⟨divp_eq_divp_of, divp_eq_divp_of⟩
+
+theorem divp_mul_divp_eq_mul_divp_mul (x y : α) (ux uy : units α) :
```

Comment:
For what it's worth, the `field` version of this is just called `div_mul_div`.

Reply:
Renamed to `divp_mul_divp`.